### PR TITLE
start and end mjd from schedule

### PR DESCRIPTION
**Pull request by mcoughlin**
**Does this pull request make any changes to the database?**
No.

Code changes that affect the database require special attention for data migration.
If your change affects the database, describe the migration plan here.

### CODE REVIEW

**Comment by shreyasahasram08** [comment]
Michael, will rebuild the db and test this shortly. Do you think it would be appropriate to also add an end time to Plan in models and Plan.html as a part of this PR? Of course it's still possible to compute the end time pretty easily as it is, but might be useful to have this when quickly glancing through multiple plans.

**Comment by mcoughlin** [comment]
Ah yes could do. If you want to add it while testing the other bit, I wouldn't complain ;)

File: growth/too/views.py [hunk 1]
```diff
@@ -1100,6 +1099,17 @@ def get_json_data(plan, decam_style=True):
     else:
         ditherNorm = 1.0
 
+    start_mjd, end_mjd = np.inf, -np.inf
```
Comment (lpsinger) [inline growth/too/views.py]: Please simplify, for example, like this:

```python
start_mjd = min(e.obstime for e in exposures).mjd
end_mjd = max(e.obstime + e.exposure_time * u.s).mjd
```

**Comment by lpsinger** [comment]
Can this be closed as a duplicate of #122, which has already been merged?

**Pull request closed without merging**